### ISSUE TITLE
Demo

### ISSUE THREAD

**Issue by medic917** [JeffreyWay/council#234]
Is there any way we can get a demo of the first release? I haven't seen anything about this on Laracast.
Excited to see a active forum project on Laravel. Not to mention from Laracast!

**Comment by divdax** [comment]
Yes, read the [readme.md](https://github.com/JeffreyWay/council/blob/master/readme.md) of the repository and install the forum.

Laracasts: https://laracasts.com/series/how-to-manage-an-open-source-project
